src/wasm.h

```diff
@@ -91,6 +91,11 @@ struct Name : public cashew::IString {
 // An index in a wasm module
 typedef uint32_t Index;
 
+inline Index toIndex(size_t index, size_t max) {
+	//assert(index < max);
```

Comment:
I removed the second parameter of toIndex() and inserted the new assert. How's it looking now for a merge?